server/drivers/postgres/index.js

```diff
@@ -29,7 +29,8 @@ const SCHEMA_SQL = `
     ns.nspname as table_schema, 
     cls.relname as table_name, 
     attr.attname as column_name,
-    trim(leading '_' from tp.typname) as data_type
+    trim(leading '_' from tp.typname) as data_type,
+    pg_catalog.col_description(attr.attrelid, attr.attnum) as col_desc
```

Comment:
What do you think about changing the field name to either `description` or `column_description`? The latter is a bit long, but I'm just thinking about the consistency in variable names...

Reply:
column_description is fine I think.  I can do that.